JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -1,55 +1,90 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
 using Amazon;
 using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using System;
-using System.Threading.Tasks;
+[assembly: SuppressMessage("Reliability", "CA2007:Do not directly await a Task", Justification = "Console application do not have a SynchronizationContext")]
```

Comment:
I think it would be best to use `<NoWarn>$(NoWarn);CA2007</NoWarn>` to do this in the project file instead.

Otherwise these assembly-wide scoped suppression are hard to find to the code files.

Reply:
ok - done